namer/database.py

```diff
@@ -0,0 +1,299 @@
+abbreviations = {
```

Comment:
my plan was to make it configurable via a map in the NamerConfig, ['aa': 'AmateurAllure]

This file would be a good seed for the config, but ultimately I have no idea what some site ripper might do, and no desire to build a list of everyone's different/conflicting fever dreams of how abbreviation should work.